samples/glean/src/main/java/org/mozilla/samples/glean/MainActivity.kt

```diff
@@ -18,11 +20,15 @@ open class MainActivity : AppCompatActivity() {
 
         Log.addSink(AndroidLogSink())
 
+        // Initialize the Glean library. Ideally, this is the first thing that
+        // must be done right after enabling logging.
+        Glean.initialize(applicationContext, Configuration())
```

Comment:
Eventually we may want to move this code from the activity to an application object to showcase the intended usage. But obviously we do not need to do that now. :)

Reply:
Ah, this is a good point. But I have no clue on how that would work. Do you think you or anyone in your team would have time to mentor/stub out the changes to the sample application? It would be great to showcase the "best" way to use the API, but I'm not sure I have the titles to say what that is :)

Reply:
I filed bug 1508227 for this.

Reply:
In a nutshell: 
* Create a class that extends `android.app.Application`: https://github.com/mozilla-mobile/reference-browser/blob/master/app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt#L13
* Reference it from the application tag (name) in AndroidManifest.xml: https://github.com/mozilla-mobile/reference-browser/blob/master/app/src/main/AndroidManifest.xml#L20
* Do you initialization from onCreate() of the application object.